insights/combiners/sudoers.py

```diff
@@ -0,0 +1,97 @@
+"""
+Sudoers - files ``/etc/sudoers`` or ``/etc/sudoers.d/*``
+========================================================
+
+Module for combining the parsing results of ``/etc/sudoers`` and
+``/etc/sudoers.d/*`` files.
+
+"""
+from insights import combiner
+from insights.parsers.sudoers import EtcSudoers
+
+
+@combiner(EtcSudoers)
```

Comment:
@xiangce Generally we develop combiner out of [two or more components](https://insights-core.readthedocs.io/en/latest/notebooks/Insights%20Core%20Tutorial.html?highlight=combiners#Combiners). Is there are special requirement because of which you decide to create a combiner out of `EtcSudoers` parser alone?

Reply:
Besides the code of `get()` & `last()` in the combiner as well as the parser is redundant.

Reply:
The `EtcSudoers` is against a `multi_output=Ture` spec, which results in the parser result being a list. This combiner is going to combine the result. 

No, they are not redundant since they work for different components.

Reply:
@psachin - if you meant the duplicate code of the two functions. I created a base class and removed the duplicates. Please have a look again.